arrays_beyza_saridas.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+def create_modified_array(d, n, m):
+    if m > n:
+        raise ValueError(" m should be less than or equal to n")
+    if d <= 0:
+        raise ValueError(" d should be greater than 0")
+    if n < 0:
+        raise ValueError(" n should be greater than 0")
+    if m < 0:
+        raise ValueError(" m should be greater than 0")
+
+    Random_array = np.random.randint(0, 10 ** d, size=(n, n))
```

Comment:
Follow the conventions for the variable names that we have mentioned in the lab.